src/vaadin-contextmenu-items-mixin.html

```diff
@@ -129,6 +132,7 @@
       return this.getAttribute('dir') === 'rtl';
     }
 
+    /** @private */
     __forwardFocus() {
```

Comment:
This is only used in `vaadin-context-menu.html`. Should be `@protected`?

Reply:
I know this opinion is far too late, but now that I think about it, it's kind of weird to decide what API we publish as protected based on how we organize our code (implementation details).

For example, we might decide to not create a separate mixin for some feature if it's really short in code and can be included in the main file or a more generic mixin. This shouldn't have anything to do with API decisions.

Reply:
Done.